ci-operator/config/kata-containers/kata-containers/wrapped-openshift-tests.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# This script requires the stock openshift-tests to be present in
+# /usr/bin/openshift-tests-original. It executes it passing all arguments
+# and only reports failures when they were not caused by invariant tests
+#
+# Primary usage is the openshift/release kata-container pipelines
+
+OUT=$(/usr/bin/openshift-tests-original $@)
+RET=$?
+
+echo "$OUT"
+
+[ "$RET" -eq 0 ] || exit 0
```

Comment:
@ldoktor shouldn't it be '&&' instead of '||'?

Reply:
Oh, yep, good catch